css/style.css

```diff
@@ -0,0 +1,997 @@
+}
+
+/* CSS Reset */
+
+/* Box sizing rules */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+/* Prevent font size inflation */
+html {
+  -moz-text-size-adjust: none;
+  -webkit-text-size-adjust: none;
+  text-size-adjust: none;
+}
+
+/* Remove default margin in favour of better control in authored CSS */
+body, h1, h2, h3, h4, p,
+figure, blockquote, dl, dd {
+  margin-block-end: 0;
+}
+
+/* Remove list styles on ul, ol elements with a list role, which suggests default styling will be removed */
+ul[role='list'],
+ol[role='list'] {
+  list-style: none;
+}
+
+/* Set shorter line heights on headings and interactive elements */
+h1, h2, h3, h4,
+button, input, label {
+  line-height: 1.1;
+}
+
+/* Balance text wrapping on headings */
+h1, h2,
+h3, h4 {
+  text-wrap: balance;
+}
+
+/* Make images easier to work with */
+img,
+picture {
+  max-width: 100%;
+  display: block;
+}
+
+/* Inherit fonts for inputs and buttons */
+input, button,
+textarea, select {
+  font-family: inherit;
+  font-size: inherit;
+}
+
+/* Make sure textareas without a rows attribute are not tiny */
+textarea:not([rows]) {
+  min-height: 10em;
+}
+
+/* Anything that has been anchored to should have extra scroll margin */
+:target {
+  scroll-margin-block: 5ex;
+}
+
+/* Site Styles Begin */
+
+body {
+  background-color: var(--white);
+  color: var(--dark-grey);
+  margin: 0;
+  overflow-x: hidden;
+  display: flex;
+  flex-direction: column;
+  font-family: 'Space Grotesk', sans-serif;
+}
+
+.is-page {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+.is-page .is-main {
+  flex: 1;
+}
+
+h1, 
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: 'Space Grotesk', sans-serif;
+}
+
+h1 {
+  font-size: var(--step-5);
+}
+
+h1 + p {
+  padding-top: 1.25rem;
+}
+
+h2 {
+  font-size: var(--step-4);
+  padding-bottom: var(--20px);
+}
+
+h3 {
+  font-size: var(--step-3);
+  padding-bottom: var(--16px);
+}
+
+h4 {
+  font-size: var(--step-2);
+  padding-bottom: var(--12px);
+}
+
+p, ul, ol {
+  max-width: 48rem;
+  line-height: 1.4;
+  font-size: var(--step-0);
+}
+
+ul, ol {
+  padding-left: 2.25rem;
+  padding-bottom: 2.25rem;
+}
+
+p + iframe {
+  margin: 2.25rem 0;
+}
+
+a {
+  color: var(--midturquoise);
+}
+
+a:hover {
+  color: var(--darkturquoise)
+}
+
+
+/* Typography Utilities */
+
+.title {
+  font-weight: 100;
+  margin: 0;
+}
+
+.subtitle {
+  font-size: var(--step-2);
+  margin-top: 0;
+  margin-bottom: var(--12px);
+}
+
+.center-text {
+  text-align: center;
+}
+
+.cta-text {
+  color: var(--turquoise);
+  margin: 0 auto;
+  padding: var(--48px) 0;
+}
+
+.header-anchor {
+  color: cadetblue;
+  width: 2.25rem;
+}
+
+/* Color Utilities */
+
+.pink {
+  color: var(--pink);
+}
+
+.orange {
+  color: var(--orange);
+}
+
+.mid-turquoise {
+  color: var(--midturquoise);
+}
+
+.green {
+  color: var(--green);
+}
+
+.yellow {
+  color: var(--yellow);
+}
+
+.gradient-back {
+  background-image: url(/img/home-gradient.png);
```

Comment:
The preview at https://servo.org/servo-redesign/ doesn't load this background image. The url is not preprocessed to be relative the PR's pathPrefix (i.e `servo-redesign`).

Reply:
Yes, this is an issue with our deployment for PRs, but I guess we can live with it. It will be showed properly once it's merged.

Reply:
Actually, there was a way to fix that. It's fixed in https://github.com/servo/servo.org/pull/202/commits/1943b5a6ca9f6712e03c2b787948cbb80a625688